### ISSUE TITLE
APN service works only if 'dry_run' is set to TRUE

### ISSUE THREAD

**Issue by Varin6** [Edujugon/PushNotification#47]
If dry_run is set to 'false', it seems to send without error, but push notification is never received in the iPhone.

**Comment by Edujugon** [comment]
Hi @Varin6 ,

What feedback are you getting? `$push->getFeedback();`

Make sure you're using the production certificate. Bear in mind that APN requires different certificates for production and testing

**Comment by Varin6** [comment]
Feedback comes as success, but yeah, I am using a development certificate, not production, which is what probably is the issue. (I'm PHP guy, someone else is building IOS app in Swift) Thanks for quick reply. Also, I'm using laravel 5.6 and your repo works fine with it,I thought you might want to know. I switched to your project from davibennun/laravel-push-notification which doesn't seem to be supporting rich push notifications for IOS at all. Really happy that I found this one!

```
["feedback":protected]=>
    object(stdClass)#1032 (3) {
      ["success"]=>
      int(1)
      ["failure"]=>
      int(0)
      ["tokenFailList"]=>
      array(0) {
      }
```
